src/utils/filterTodo.ts

```diff
@@ -0,0 +1,24 @@
+import { ITodoFilter, Todo } from '../types/Todo';
+import { FilterStatuses } from './enums/FiltersStatus';
+
+export const getFiltredTodo = (
+  todos: Todo[],
+  { status, filter }: ITodoFilter,
+) => {
+  let filteredTodos = todos.filter(({ title }) =>
+    title.toLowerCase().trim().includes(filter),
+  );
+
+  filteredTodos = filteredTodos.filter(({ completed }) => {
+    switch (status) {
+      case FilterStatuses.Active:
+        return !completed;
+      case FilterStatuses.Completed:
+        return completed;
+      default:
+        return true;
+    }
+  });
```

Comment:
I think you can combine it into one filter, and split it into two functions in the middle.
like: 

```suggestion
  
  const isTitleMatchedToFilter = (title) => title.toLowerCase().trim().includes(filter);
  
  const filterByStatus = (status, completed) => {
    switch (status) {
      case FilterStatuses.Active:
        return !completed;
      case FilterStatuses.Completed:
        return completed;
      default:
        return true;
    }
  }

  return todos.filter(({ completed, title }) => {
    if(!isTitleMatchedFilter(title) {
      return;
    }
    
    return filterByStatus(status, completed); 
   
  });
```

Reply:
I understand,so if i have 10 filters it would be better to create more function to check and for filtered than create a huge ternary operate